content/en/docs/concepts/architecture/self-healing.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Kubernetes Self-Healing  
+content_type: concept  
+Weight: 50  
+---
+<!-- overview -->
+
+Kubernetes is designed with self-healing capabilities that help maintain the health and availability of workloads. 
+It automatically replaces failed containers, reschedules workloads when nodes become unavailable, and ensures that the desired state of the system is maintained.
+
+<!-- body -->
+
+## Self-Healing capabilities {#self-healing-capabilities} 
+
+- **Pod Restart:** If a container inside a Pod fails, Kubernetes restarts it based on the [`restartPolicy`](/docs/concepts/workloads/pods/pod-lifecycle/#restart-policy)`. 
+
+- **Replica replacement:** If a Pod in a [`Deployment`](/docs/concepts/workloads/controllers/deployment/) or [`ReplicaSet`](/docs/concepts/workloads/controllers/replicaset/) fails, Kubernetes creates a replacement Pod to maintain the specified number of replicas. 
```

Comment:
- **Replica replacement:** If a Pod in a [Deployment](/docs/concepts/workloads/controllers/deployment/) or [StatefulSet](/docs/concepts/workloads/controllers/statefule/) fails, Kubernetes creates a replacement Pod to maintain the specified number of replicas.
  If a Pod fails that is part of a [DaemonSet](/docs/concepts/workloads/controllers/daemonset/), the control plane
  creates a replacement Pod to run on the same node.
```